### PR TITLE
Add conclusion property to CheckRun

### PR DESCRIPTION
**Pull request by KristinBi**
Added the conclusion property to be a member variable of checkRun class. 
Solved the https://github.com/flutter/flutter/issues/99850 issue.

### CODE REVIEW

**Comment by KristinBi** [comment]
Hi @CaseyHillers, could please help take a look? This is blocking https://github.com/flutter/flutter/issues/99850 for the sutosubmit project. Thanks!

File: lib/src/common/model/checks.dart [hunk 1]
```diff
@@ -127,6 +129,7 @@ class CheckRun {
       checkSuiteId: input['check_suite']['id'],
       detailsUrl: input['details_url'],
       startedAt: DateTime.parse(input['started_at']),
+      conclusion: CheckRunConclusion._fromValue(input['conclusion']),
```
Comment (CaseyHillers) [inline lib/src/common/model/checks.dart:132]: Please add a test

Comment (KristinBi) [reply lib/src/common/model/checks.dart:132]: Hi Casey, I didn't find any unit test for this check file, do you want me to add a unit test for this class?

Comment (CaseyHillers) [reply lib/src/common/model/checks.dart:132]: Yes, please add the unit test

Comment (KristinBi) [reply lib/src/common/model/checks.dart:132]: Sure, I will add one now.

File: test/unit/checks_test.dart [hunk 2]
```diff
@@ -0,0 +1,30 @@
+import 'dart:convert';
+
+import 'package:github/src/common/model/checks.dart';
+import 'package:test/test.dart';
+
+void main() {
+  group('Check run', () {
+    test('CheckRun fromJson', () {
+      // This is a truncated version of the response
+      const checkRunJson = '''{
```
Comment (CaseyHillers) [inline test/unit/checks_test.dart:10]: Please use the official GitHub values provided in the API -- https://docs.github.com/en/rest/reference/checks#get-a-check-run

And add a link to where it is from

Comment (KristinBi) [reply test/unit/checks_test.dart:10]: Sure Casey, I will change this to the official one.

**Comment by KristinBi** [comment]
Thank you Casey for your help as always!

**Comment by CaseyHillers** [comment]
@robbecker-wf can you publish this?

**Comment by robrbecker** [comment]
I can publish in the morning, just heading to bed.

**Comment by robrbecker** [comment]
published!

**Pull request merged**